releasetool/commands/tag/php.py

```diff
@@ -21,9 +21,8 @@ def kokoro_job_name(upstream_repo: str, package_name: str) -> Union[str, None]:
 
     if repo_short_name == "google-cloud-php":
         return "cloud-devrel/client-libraries/php/google-cloud-php/docs/docs"
```

Comment:
tangentially related - should we change this to `"cloud-devrel/client-libraries/php/google-cloud-php/docs`? This was done accidentally by me (I didn't realize putting it in a `docs` directory would have this effect)

Reply:
Created cl/549033328 for this.